src/olympia/applications/tests.py

```diff
@@ -83,3 +84,32 @@ def test_addnewversion(self):
 
         assert len(AppVersion.objects.filter(
             application=amo.FIREFOX.id, version=new_version)) == 1
+
+    @mock.patch('olympia.applications.management.commands.import_prod_versions'
+                '.PyQuery', spec=True)
```

Comment:
Was also wtfing. It's purely for local development purposes though... I didn't even know PyQuery could fetch a url directly...